scraibe/diarisation.py

```diff
@@ -183,7 +184,7 @@ def _save_token(token):
     
     @classmethod
     def load_model(cls, 
-                    model: str = PYANNOTE_DEFAULT_CONFIG, 
+                    model: str = PYANNOTE_FALLBACK_CONFIG, 
                     use_auth_token: str = None,
                     cache_token: bool = True,
```

Comment:
Set this by default to false due to security concerns

Reply:
@mahenning set cache_token to False

Reply:
Fixed this in [55a77b8](https://github.com/JSchmie/ScrAIbe/pull/71/commits/55a77b861cf40e5354e719e511ebe0e6977fe212).